test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -603,15 +603,19 @@ private boolean isNativeTest() {
         return false;
     }
 
+    private boolean matchingQuarkusTestProfileTags(ExtensionContext context) {
```

Comment:
Is this used anywhere? It does not look right

Reply:
Fixed